lib/ffi/clang/lib/cursor.rb

```diff
@@ -232,7 +232,8 @@ module Lib
 				:cursor_omp_depobj_directive, 286,
 				:cursor_omp_scan_directive, 287,
 				# :cursor_last_stmt, :cursor_omp_scan_directive,
-				:cursor_translation_unit, 350,
+        # In Clang 15 the enum value changed from 300 to 350!
+				:cursor_translation_unit, Clang.clang_version < Gem::Version.new('15.0.0') ? 300 : 350,
```

Comment:
Can we extract this into a constant?

e.g.

```ruby
# In Clang 15 the enum value changed from 300 to 350!
CUSOR_TRANSLATION_UNIT = Clang.clang_version < Gem::Version.new('15.0.0') ? 300 : 350
```

Then use it later in the enum? Just to keep things a little more streamlined.

Reply:
Actually that was how I first implemented it. But then I thought you might not like having the constant. So sure, I'll update the PR.